node_modules/@stdlib/iter/cartesian-power/docs/repl.txt

```diff
@@ -0,0 +1,36 @@
+{{alias}}( x, n )
+    Returns an iterator which generates the Cartesian power of an input array-like object.
+
+    If an environment supports Symbol.iterator, the returned iterator is iterable.
+
+    Parameters
+    ----------
+     x : ArrayLike<T> | Iterable<T>
```

Comment:
This is not TypeScript. Please read https://github.com/stdlib-js/stdlib/blob/develop/docs/repl_text.md